package-lock.json

```diff
@@ -16678,7 +16678,7 @@
       "integrity": "sha512-m62ShEObQ39CfralilEQRjH6oAMtNCV1xJyEx5LpRYUVN+EviphDgUc/F3hnYbADmkiNs67Y+3ylmlG7Lnu+FA==",
       "dev": true,
       "requires": {
-        "es5-ext": "0.10.53",
```

Comment:
Normally this would be fine, but I'm hesitant to change `package-lock.json` between release candidates. Can you revert the changes to this file and check if we still have a passing build? I don't see why changes to the webpack configuration would require `package-lock.json` changes.

Reply:
Testing.

Reply:
I'm attempting to re-run it, but it seems that the macos build fails.

Reply:
Re-ran, it passed.